docs/DeveloperGuide.md

```diff
@@ -274,6 +278,12 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
       
+* 1d. User forgets to indicate the priority of the client. 
+
+    * 1d1. System saves the client with an undefined priority. 
```

Comment:
Perhaps we can say System adds a new client instead of "saves"
I'm also not sure if this is a valid/relevant 1d1 as in the positive use case MSS, we do not mention the System adding/saving clients. I think we can either 
  1. Include add/save client in the use case MSS (but that means changing other use cases as well)
  2. Adjust 1d1 so it sounds consistent as the others
  3. Not have this extension as resolving might be hard and it's on a different level than the use case

Reply:
yes ok then I think we should wait on the one javier has done before I include this, in the meantime, ill delete this